tests/FileFormat/PdfStatisticsTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Tests\CubeTools\CubeCommonBundle\FileFormat;
+
+use CubeTools\CubeCommonBundle\FileFormat\PdfStatistics;
+use PHPUnit\Framework\TestCase;
+
+class PdfStatisticsTest extends TestCase
+{
+    protected $folderWithPdfsForTests = 'tests' . DIRECTORY_SEPARATOR . 'Resources' . DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR;
```

Comment:
Note: php converts to '\' on windows magically. (Or / works on windows, at least it does in cmd and file explorer.)
But maybe good style?   https://secure.php.net/manual/de/dir.constants.php#113343